services/ui-src/src/components/app/App.test.tsx

```diff
@@ -7,34 +7,31 @@ import {
   mockNoUser,
   RouterWrappedComponent,
 } from "utils/testing/setupJest";
-import { useUser } from "utils/auth";
 //components
 import { App } from "components";
 
-jest.mock("utils/auth");
-const mockedUseUser = useUser as jest.Mock<typeof useUser>;
+jest.mock("utils/auth", () => ({
+  useUser: jest
+    .fn()
+    .mockReturnValueOnce(mockStateUser)
```

Comment:
This will depend on test execution order. When mock setup varies between test, it is better to do the setups within the individual tests themselves.

Reply:
It was that way before. Tests were actually passing with the changes, but typescript didn't like it
I'll give it another shot